php/libraries/User.class.inc

```diff
@@ -66,26 +66,41 @@ class User extends UserPermissions
         $row = $DB->pselectRow($query, array('UID' => $username));
```

Comment:
should there be a check to see if $username is actually set before setting / retrieving data above?